resources/light/s3/create-bucket.svg

```diff
@@ -0,0 +1,15 @@
+        <polygon id="Path" fill="#424242" points="4.13501792 7.08891292 3.13501792 7.08891292 3.13501792 4.08891292 0.135017917 4.08891292 0.135017917 3.08891292 3.13501792 3.08891292 3.13501792 0.0889129225 4.13501792 0.0889129225 4.13501792 3.08891292 7.13501792 3.08891292 7.13501792 4.08891292 4.13501792 4.08891292"></polygon>
```

Comment:
Matches the VSCode color scheme. 
It's quite noticeable on the light color scheme.

The blue drowns out the gray.
![image](https://user-images.githubusercontent.com/1410453/83080278-a4af9200-a032-11ea-8ea8-61b118c1fb5b.png)

versus

![image](https://user-images.githubusercontent.com/1410453/83080334-c446ba80-a032-11ea-94e6-f65a9353c769.png)

I will update the other icons to match this in a future PR since this bothers me 😄 .